### PR TITLE
Added docs/cloud toggle on the first few steps in build-your-first-mo…

### PR DESCRIPTION
**Pull request by breezyfasano**
## Description & motivation
See [this Slack convo](https://dbt-labs.slack.com/archives/C01C2H652BC/p1634316063261500
) for context.
**TL;DR:** I added Cloud/Core toggle functionality to the first few steps of the process explained on `/tutorial/build-your-first-models` where previously, it was formatted with just Markdown headers.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [X] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

@JKarlavige I just want to call out that when I ran `npm install` inside `website`, it updated the `package-lock.json` and it's going to be committed in this PR. I think it should be fine, but let me know if not!

### CODE REVIEW

**Review by JKarlavige — APPROVED** [review]
lgtm!

**Comment by JKarlavige** [comment]
> I just want to call out that when I ran `npm install` inside `website`, it updated the `package-lock.json` and it's going to be committed in this PR. I think it should be fine, but let me know if not!

To follow up on this. The package-lock is restructured for me every time I run npm-install. It looks like it only restructures the file, while the dependencies still have the same version and the site behaves the same. So I don't see this as an issue.

**Comment by annafil** [comment]
Looks great ✅  thank you for fixing this so quickly ✨
